src/test/kotlin/com/ampnet/mailservice/service/MailServiceTestBase.kt

```diff
@@ -136,6 +142,24 @@ abstract class MailServiceTestBase : TestBase() {
             .setOwner(owner)
             .build()
 
+    private fun generateMailResponse(
+        coop: String,
+        type: MailType,
+        id: Int? = null,
+        title: String = "mail title",
+        lang: Lang = defaultLanguage
+    ): MailResponse {
+        val requiredFields = type.getRequiredFields().map { it.value }
+        return MailResponse(
+            id, coop, title, requiredFields.joinToString(),
+            type, requiredFields, lang
+        )
+    }
```

Comment:
```suggestion
    private fun generateMailResponse(
        coop: String,
        type: MailType,
        id: Int? = null,
        lang: Lang = defaultLanguage
    ): MailResponse {
        val requiredFields = type.getRequiredFields().map { it.value }
        return MailResponse(
            id, coop, type.name, requiredFields.joinToString(),
            type, requiredFields, lang
        )
    }
 
```
This way you can assert mail title/subject in the tests.